FrontEnd/src/components/HeaderFooter/header/navbar/SearchBox.module.css

```diff
@@ -27,21 +27,20 @@
 }
 
 .header-search-form__addon {
-    display: flex;
-    padding: 9px;
-    width: 14px;
-    height: 14px;
-    justify-content: center;
-    align-items: center;
-    gap: 8px;
-    align-self: stretch;
-    border-left: 1px solid #D9D9D9;;
+    background-color: transparent;
+    border: none;
```

Comment:
Do we need these two if we define border two lines bellow?

Reply:
Fixed